docs/rules/use-ember-get-and-set.md

```diff
@@ -24,24 +24,25 @@ To be auto-fixed, `Ember` must be imported.
 Ideally, you should also be using [local modules](./local-modules.md); otherwise, the fixed code will look like `Ember.get(this, fooProperty')` instead of `get(this, 'fooProperty')`.
 
 ```javascript
-// Bad
+// Unrecommended
```

Comment:
This sounds weird. Perhaps "Not recommended"?

Reply:
✅